lib/rules/prefer-stateless-function.js

```diff
@@ -298,6 +302,20 @@ module.exports = {
         markThisAsUsed(node);
       },
 
+      // Report non-"render" methods that return JSX
+      MethodDefinition: function(node) {
```

Comment:
what about computed methods? like `['foo']() { return <div /> }` or `[Symbol()]() { return <div />; }`?

Reply:
TBH I'm not sure why this commit was in the original branch that I started from here. I'd be fine just removing that commit from this PR, but I'm not sure what would be correct in regards to your comment here.

Reply:
ah yes, good call. This PR is about the no-typos rule; but it has prefer-stateless-function changes. Let's remove all the parts that aren't about no-typos :-)